Cesium.CodeGen/Contexts/TranslationUnitContext.cs

```diff
@@ -1,8 +1,9 @@
+using System.Reflection;
 using Mono.Cecil;
 
 namespace Cesium.CodeGen.Contexts;
 
-public record TranslationUnitContext(AssemblyContext AssemblyContext)
+public record TranslationUnitContext(AssemblyContext AssemblyContext, Assembly[] ImportAssemblies)
```

Comment:
The idea is good, but could you please move that list up to the `AssemblyContext`?

I am already preparing the architecture for the case when there are multiple translation units in a single assembly.